src/qtoolkit/core/data_objects.py

```diff
@@ -171,7 +171,7 @@ class QResources(QTKObject):
     njobs: int | None = None
     """Number of jobs in a job array."""
 
-    kwargs: dict | None = None
+    qkwargs: dict | None = None
```

Comment:
Thinking about the name, maybe scheduler_kwargs ? more verbose but basically it is keyword arguments to be passed to a baseschedulerIO instance right ?